castle/cms/upgrades/upgrade_2_1_0.py

```diff
@@ -9,7 +9,7 @@
 def upgrade(context, logger=None):
     # This value is being converted to a tuple, hold onto it or it will get
     # overwritten
-    backend_url = api.portal.get_registry_record('plone.backend_url') or ''
+    backend_url = api.portal.get_registry_record('plone.backend_url') or u''
```

Comment:
I think more proper thing here is `or None`.

We don't want a value of `(u'',)` either.

Reply:
I'll buy that, I got hung up on the unicode bit...

Reply:
so then, I think you should set the value to `()` when None.